dm/config/source_config.go

```diff
@@ -232,7 +232,8 @@ func (c *SourceConfig) Adjust(ctx context.Context, db *sql.DB) (err error) {
 		}
 	}
 
-	if c.EnableGTID {
+	// MariaDB automatically enabled gtid after 10.0.2, refer to https://mariadb.com/kb/en/gtid/#using-global-transaction-ids
```

Comment:
I guess there's no way to disable master's GTID right 🤔

Reply:
Yes, I can't find a way to disable it.